docs/components/MessagingAvatarView.jsx

```diff
@@ -50,6 +50,12 @@ export default class MessagingAvatarView extends React.PureComponent {
               text={convertNameToInitials("Kristen Stark")}
               color={{ seed: "52c5e9d0e6ed0694212d69ed" }}
             />
+            <br />
+            <MessagingAvatar
+              text={convertNameToInitials("Dewey the Fox")}
```

Comment:
Doesn't matter cause we have an image anyway, but I might suggest different initials. 😅